backend/entityservice/tasks/base_task.py

```diff
@@ -105,7 +105,16 @@ def celery_bug_fix(*args, **kwargs):
 
 @celery.task(base=BaseTask, ignore_result=True)
 def on_chord_error(*args, **kwargs):
-    logger.info("An error occurred while processing the chord's tasks")
+    """
+    Record that a task has encountered an error, mark the run as failed.
+
+    :param args: A 1-tuple starting with the taskid.
+    :param kwargs: Keyord arguments to the task e.g. {'run_id': '...', }
+    """
+    task_id = args[0]
```

Comment:
according to this: https://github.com/celery/celery/issues/3709#issuecomment-407491389 The error task gets called only with a result id which corresponds to the `ChordError` result.
Maybe we can get hold of that result and maybe maybe it contains some useful information....

Reply:
that comment may be refering to a different version or is at least partially wrong. The error handler's `kwargs` component has the task's arguments - which I use to extract `run_id`. Feel free to look into it separately though.